### ISSUE TITLE
Add mumbai chain config

### ISSUE THREAD

**Issue by Stentonian** [Float-Capital/config#6]

**Comment by JonoPrest** [comment]
@Stentonian is this for the rescript client?

**Comment by Stentonian** [comment]
@JonoPrest yes, or for dapp, or for anyone that wants to see details of our contracts

**Comment by JonoPrest** [comment]
> @JonoPrest yes, or for dapp, or for anyone that wants to see details of our contracts

Cool perfect! Was doing a backlog grooming. I was just checking that it's still relevant and for the npm package.
